store/store.js

```diff
@@ -0,0 +1,111 @@
+const encoding = require('../encoding/encoding');
+const Session = require('../session/session');
+
+module.exports = class Store {
+
+  #cache;
+  #secret;
+
+  static #idOctets = 21;
+  static #signatureStart = 28;
+  static #signatureLength = 27;
+  static #valueLength = signatureStart + signatureLength;
+
+  constructor() {
+    //TODO : Source cache details from config, or passed into store?
```

Comment:
Can there be both?
```
constructor(password = process.env.PW_VAR, addr = process.env.ADDR_VAR, db = process.env.DB_VAR) {
  this.cache = new Cache({
    password,
    addr,
    db
  })
}
```
Will probably make for an easier test, gives valid default behaviour and doesnt restrict anyone from trying something different